components/textbook-beta/StartLearningSection.vue

```diff
@@ -6,7 +6,7 @@
     <article
       v-for="{ title, description, courses } in learningSections"
       :key="title"
-      class="bx--row"
+      class="bx--row start-learning-section__row"
```

Comment:
Could you double check this? I think you don't need this `start-learning-section__row` because we are already using the carbon styles for rows

Reply:
For this, I added a class that _we can own_, because we need to add space between these "sections"

<img width="1680" alt="Screen Shot 2021-09-14 at 5 54 29 AM" src="https://user-images.githubusercontent.com/6276074/133261762-42c71673-7ec0-46e8-aef5-38a284bb7a61.png">

I then use that class to add spacing, [here](https://github.com/Qiskit/qiskit.org/pull/2157/files#diff-4e893dec92fbbb19f41f5afab4139ffcec5f88324e74eae9a1369f7bda8511feR174)


Let me know if there's a better way